Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,825 @@
+    end
+```
+
+## Required Build Inputs
+
+When building feature bands, you'll work with two types of source-built
+artifacts:
+
+### Previously Source-Built (PSB) Artifacts
+
+These are artifacts from previous servicing releases or bootstrap builds:
+
+- Used as build dependencies but not redistributed in outputs
+- Come from previous monthly .NET releases (e.g., 8.0.1 artifacts when
+  building 8.0.2)
+- Subject to [poison detection](leak-detection.md) to ensure no Microsoft
+  binaries leak into outputs
+
+### Shared Component Artifacts
+
+These are shared runtime and foundational components from the 1xx band:
+
+- Come from the 1xx band build in the same release cycle
+- Can be used as dependencies AND redistributed in outputs
+- Include shared runtime and foundational components that all feature bands
+  depend on
+
+## Feature Band Characteristics
+
+### 1xx Band (GA/Non-breaking)
+
+The foundational feature band that ships with .NET GA releases:
+
+- **Content**: Contains the complete VMR with .NET runtime, core libraries,
+  and SDK tooling
+- **Purpose**: Provides the stable runtime foundation that all other bands
+  depend on
+- **Release cycle**: Follows .NET GA release schedule with monthly servicing
+  updates
+- **Compatibility**: Non-breaking changes only during servicing
+
+### 2xx Band (Stable/Enhanced)
+
+Enhanced SDK tooling that ships with Visual Studio updates:
+
+- **Content**: Contains only SDK tooling differences from 1xx band (subset
+  VMR)
+- **Purpose**: Adds enhanced developer tooling and Visual Studio integration
+  features
+- **Release cycle**: Aligns with Visual Studio feature releases (not
+  necessarily monthly)
+- **Compatibility**: Stable features that have been validated in 3xx previews
+- **Dependencies**: Always uses shared runtime components from 1xx band
+
+### 3xx Band (Preview/Insiders)
+
+Cutting-edge SDK features for early adopters and insiders:
+
+- **Content**: Contains only SDK tooling differences from 2xx band (subset
+  VMR)
+- **Purpose**: Delivers latest experimental features and early access
+  capabilities
+- **Release cycle**: Independent preview releases, not aligned with other
+  bands
+- **Compatibility**: Preview/experimental features that may change before
+  stabilization
+- **Dependencies**: Uses 2xx tooling for builds, shares runtime from 1xx band
+- **Flow**: Selected features flow from 3xx → 2xx over time through
+  cherry-picking
+
+## Build Requirements by Feature Band
+
+Each feature band has specific requirements for SDKs and artifacts:
+
+### 1xx Band Build Requirements
+
+- **Bootstrap (any version)**: Two-stage process using Microsoft source-built
+  1xx artifacts + Microsoft SDK + script
+- **Servicing (N.0.101+)**: Source-built SDK and artifacts from the previous
+  1xx release
+
+Note: Only the 1xx SDK is guaranteed to build the shared runtime components
+
+### 2xx Band Build Requirements
+
+- **Bootstrap (any version)**: Two-stage process using Microsoft source-built
```

Comment:
I think this needs more explanation and shouldn't be listed at the first item.  This is really an exceptional use case if MSFT screws up and needs to rebootstrap in servicing or if a customer wants to start building a servicing release without source building all previously released versions.